src/mixer/basetrackplayer.cpp

```diff
@@ -805,34 +822,29 @@ void BaseTrackPlayerImpl::slotSetTrackColor(const mixxx::RgbColor::optional_t& c
     m_pTrackColor->forceSet(trackColorToDouble(color));
 }
 
-void BaseTrackPlayerImpl::slotTrackColorPrev(double v) {
+void BaseTrackPlayerImpl::slotTrackMoveColor(int steps) {
     if (!m_pLoadedTrack) {
         return;
     }
 
-    if (v > 0) {
-        ColorPaletteSettings colorPaletteSettings(m_pConfig);
-        ColorPalette colorPalette = colorPaletteSettings.getTrackColorPalette();
-        mixxx::RgbColor::optional_t color = m_pLoadedTrack->getColor();
-
-        color = colorPalette.previousColor(color);
-        m_pLoadedTrack->setColor(color);
-    }
-}
-
-void BaseTrackPlayerImpl::slotTrackColorNext(double v) {
-    if (!m_pLoadedTrack) {
+    if (steps == 0) {
```

Comment:
Please convert the steps to integers with std::abs or static_cast<int>

It might be possible to set the CO to 0.5 which will trigger an infinite loop IIUC

Reply:
The method signature only accepts integers :
`void BaseTrackPlayerImpl::slotTrackMoveColor(int steps)`